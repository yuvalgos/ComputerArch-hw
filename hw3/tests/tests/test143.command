./dflow_calc tests/test143.dat tests/test143.in d132 d176 p17 p40 p50 p105 d154 d91 d18 p147 p151 d57 p112 d97 p15 d88 d135 d30 p169 d146 p10 d23 d5 p143 d152 d185 d101 p145 d88 d62 p54 d127 d111 p178 d149 d21 p142 d52 p177 p61 p18 d0 d82 d121 p106 p83 p93 d142 d168 p176 d182 p2 d121 d64 d153 d31 p161 p100 d131 p82 p175 p34 p45 p198 p91 d29 d36 d108 d99 p158 d70 p4 p168 d128 d34 p55 d190 p85 p182 d63 p168 d1 d30 d171 d145