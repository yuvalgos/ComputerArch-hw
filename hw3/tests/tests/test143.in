4 31 7 20
1 9 26 21
7 12 18 15
1 25 5 30
2 7 31 7
7 24 1 8
1 30 28 26
5 17 7 7
5 14 6 17
1 20 30 3
3 28 10 7
2 30 31 16
3 12 22 18
3 1 21 9
4 26 13 22
7 19 12 2
5 25 27 14
0 0 2 1
8 21 30 11
1 29 17 27
2 20 21 9
3 31 5 15
1 14 7 1
5 17 7 8
5 5 5 6
3 12 28 26
3 10 5 19
4 21 7 16
3 4 18 29
2 30 20 3
6 22 7 3
2 3 21 6
3 9 11 7
6 6 19 17
2 2 24 29
2 31 31 12
3 27 25 30
3 10 8 30
3 16 6 3
1 13 25 5
4 12 30 19
7 26 29 7
0 15 27 7
7 0 26 12
1 0 8 28
8 1 28 11
6 27 17 9
2 28 12 1
5 24 20 5
1 2 5 20
7 3 14 22
6 8 12 15
5 29 15 2
3 4 19 31
2 28 14 22
8 1 11 27
6 7 28 22
0 26 14 22
7 11 21 26
0 7 11 11
6 24 24 3
6 15 7 1
4 23 4 23
6 16 16 25
8 29 2 1
3 28 24 18
3 29 5 12
3 27 26 28
8 13 13 13
7 22 6 24
6 26 15 7
6 20 11 30
4 10 19 18
8 4 11 3
4 22 20 23
1 18 17 30
8 25 16 15
2 1 27 9
4 27 21 29
0 9 15 4
2 27 0 26
4 7 19 12
4 15 14 7
3 12 8 23
3 5 22 11
6 20 21 27
7 30 0 8
0 11 12 28
2 19 23 5
7 22 16 10
0 17 15 8
8 24 8 31
0 25 30 17
6 21 31 29
6 16 11 16
1 20 17 6
4 29 4 21
8 13 3 2
3 24 26 27
6 22 5 1
3 2 1 26
5 11 20 24
8 21 16 0
1 15 21 10
1 11 30 5
6 19 16 3
3 20 2 14
5 3 0 0
1 22 1 6
7 7 30 10
3 12 30 31
7 14 7 18
6 31 8 23
5 5 21 24
6 26 31 24
1 31 24 3
5 14 26 15
6 1 20 4
2 11 21 6
6 0 27 17
4 10 19 23
2 11 21 30
1 10 20 27
0 6 3 15
1 7 12 12
2 28 16 8
0 4 9 17
6 9 28 17
2 2 7 29
3 6 31 18
8 27 15 0
7 20 18 27
1 28 28 26
7 17 4 25
6 12 12 28
0 1 12 3
1 12 24 26
5 25 15 20
7 8 12 20
1 22 14 28
4 24 0 4
1 7 18 1
4 28 5 17
4 30 18 23
7 8 21 8
5 0 19 19
7 9 12 7
6 28 9 30
5 17 15 0
7 11 5 27
5 25 22 5
8 20 0 19
6 27 11 14
7 20 20 11
5 14 27 25
3 17 17 4
1 28 18 11
5 11 0 4
5 14 16 4
4 22 25 18
5 24 18 11
0 28 18 7
5 29 11 25
3 1 20 12
1 13 20 20
7 27 1 14
4 30 1 13
4 31 21 11
1 29 29 19
7 10 0 24
4 19 21 17
0 25 3 5
1 0 14 2
3 27 25 23
0 2 31 14
5 3 3 16
1 15 7 13
2 25 5 24
7 5 26 25
2 17 11 27
3 7 30 27
7 27 9 27
0 18 10 20
1 14 18 10
3 15 18 30
2 12 7 10
1 7 27 20
8 12 18 3
6 2 7 19
2 23 30 5
4 10 3 13
2 7 5 31
5 29 9 20
6 14 16 19
8 30 2 24
6 25 27 30
7 31 14 28
6 9 3 6
8 10 20 29
0 1 18 28
