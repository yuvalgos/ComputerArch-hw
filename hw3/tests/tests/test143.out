Reading the opcodes latency info from tests/test143.dat ... Got latency for 9 opcodes
Reading the program file tests/test143.in ... Found 200 instructions
getProgDepth()==255
getInstDeps(132)=={125,114}
getInstDeps(176)=={141,164}
getDepDepth(17)==10
getDepDepth(40)==49
getDepDepth(50)==33
getDepDepth(105)==107
getInstDeps(154)=={152,150}
getInstDeps(91)=={51,35}
getInstDeps(18)=={11,-1}
getDepDepth(147)==176
getDepDepth(151)==137
getInstDeps(57)=={22,30}
getDepDepth(112)==81
getInstDeps(97)=={50,49}
getDepDepth(15)==7
getInstDeps(88)=={62,84}
getInstDeps(135)=={134,107}
getInstDeps(30)=={4,-1}
getDepDepth(169)==163
getInstDeps(146)=={136,141}
getDepDepth(10)==16
getInstDeps(23)=={4,-1}
getInstDeps(5)=={-1,-1}
getDepDepth(143)==53
getInstDeps(152)=={149,116}
getInstDeps(185)=={180,169}
getInstDeps(101)=={95,98}
getDepDepth(145)==124
getInstDeps(88)=={62,84}
getInstDeps(62)=={53,-1}
getDepDepth(54)==33
getInstDeps(127)=={125,90}
getInstDeps(111)=={109,75}
getDepDepth(178)==168
getInstDeps(149)=={113,130}
getInstDeps(21)=={-1,-1}
getDepDepth(142)==143
getInstDeps(52)=={42,49}
getDepDepth(177)==163
getDepDepth(61)==69
getDepDepth(18)==16
getInstDeps(0)=={-1,-1}
getInstDeps(82)=={22,81}
getInstDeps(121)=={102,86}
getDepDepth(106)==110
getDepDepth(83)==81
getDepDepth(93)==67
getInstDeps(142)=={113,133}
getInstDeps(168)=={162,105}
getDepDepth(176)==197
getInstDeps(182)=={169,153}
getDepDepth(2)==0
getInstDeps(121)=={102,86}
getInstDeps(64)=={49,55}
getInstDeps(153)=={151,149}
getInstDeps(31)=={27,-1}
getDepDepth(161)==157
getDepDepth(100)==103
getInstDeps(131)=={75,130}
getDepDepth(82)==71
getDepDepth(175)==107
getDepDepth(34)==32
getDepDepth(45)==23
getDepDepth(198)==207
getDepDepth(91)==81
getInstDeps(29)=={20,-1}
getInstDeps(36)=={16,29}
getInstDeps(108)=={77,33}
getInstDeps(99)=={84,77}
getDepDepth(158)==107
getInstDeps(70)=={61,59}
getDepDepth(4)==11
getDepDepth(168)==175
getInstDeps(128)=={124,96}
getInstDeps(34)=={5,19}
getDepDepth(55)==46
getInstDeps(190)=={175,164}
getDepDepth(85)==108
getDepDepth(182)==194
getInstDeps(63)=={38,16}
getDepDepth(168)==175
getInstDeps(1)=={-1,-1}
getInstDeps(30)=={4,-1}
getInstDeps(171)=={107,113}
getInstDeps(145)=={105,105}
